### ISSUE TITLE
tmux?

### ISSUE THREAD

**Issue by jgallen23** [progrium/envy#35]
I normally use tmux when I'm in my dev environment so I can run the web server in one window, run grunt in another and vim in another.  I've noticed that if I don't kill the tmux session and exit ssh, the processes are still running, but I can't reattach when I log in again.  I've tried storing the tmux socket in a shared directory, but that doesn't work either.  How are others handling multiple terminals?

**Comment by progrium** [comment]
My general thought was that persistent processes should be done via Docker. But for what tmux does (wraps current shell session) I'm not sure that will work. I don't really want to support stateful/persistent sessions. So figuring out how to make it work with Docker is what I'd suggest. Your user Docker might need to run tmux and SSH back into Envy. And then every time you access it you'd connect to tmux socket from your environment. This probably requires more volume sharing with the user Docker.

**Comment by jgallen23** [comment]
Makes sense.  I've been doing tmux for awhile, but I've been wanting to look into docker compose, so maybe now's the time

**Comment by progrium** [comment]
Couldn't you do tmux from your laptop and have each terminal open an envy session?
